lib/helper/Playwright.js

```diff
@@ -250,11 +251,17 @@ class Playwright extends Helper {
     try {
       requireg('playwright');
     } catch (e) {
-      return ['playwright@^0.10'];
+      return ['playwright@^0.12.1'];
     }
   }
 
-  _init() {
+  async _init() {
+    // register an internal selector engine for reading value property of elements in a selector
+    try {
+      await playwright.selectors.register('__value', createValueEngine);
+    } catch (e) {
+      console.warn(e);
```

Comment:
out of curiosity, why we just give a warning in log here?

Reply:
This is something I wanted to check actually, how do I do error handling in CodeceptJS wasn't to avoid unhandled rejection